com.github.giacomogroppi.writernote-qt.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--share=network",
```

Comment:
does it need network access?

Reply:
yes because there is a module that alerts the user when there is a new update to be done

Reply:
so it doesn't need. since only flatpak can update.

Reply:
Yeah, I suggest dropping it. You can use https://github.com/flathub/flatpak-external-data-checker/ to automate updates on Flathub's end.